packages/tools/client-debugger/client-debugger/src/FluidDebuggerLogger.ts

```diff
@@ -98,7 +98,7 @@ export class FluidDebuggerLogger extends TelemetryLogger {
 	public static create(
```

Comment:
If we keep the pattern of the consumer being responsible for instantiating this, maybe I'm ok with this change, because when they give it to dev tools initialization we might need to access more functionality on it. Originally the idea was that the consumer doesn't need to know the details about what this logger can do, they just need something that complies with the interface they have to provide to the loader (which in fact should have been `ITelemetryBaseLogger`)

If we make it so the dev tools initialization is how the consumer gets the logger, then I think we definitely stick with `ITelemetryBaseLogger`, while internally treat it as the actual type so we can use whatever other functionality we put in there (like I imagine we might end up needing, to check for capabilities).

Reply:
They're still calling the static constructor on the specific type either way, though, so I guess I don't see the harm in using the specific type regardless. If they didn't know the type at all, then I would totally agree, but in this case they do.

Reply:
Yeah, for this PR it makes sense. You did sell me on making the logger a return value from initializeDevTools though 😄. Which would mean we always enable the telemetry view...? Or add a param to that init function to say if you want telemetry or not, and you get an object back that has (or not) the logger depending on your choice? Feels weirder than just enabling the feature all the time.

Reply:
Hmm, that's an interesting point. I had been envisioning the app passing in their own logger, and we would construct a mixin logger from it, but that doesn't make sense for at least a couple of reasons. This probably merits a bit more discussion.

Reply:
Yeah. One of the approaches in the proposal for message latency even lets DevTools inject the logger it needs without the user having to do anything at all. So more discussion sounds good.